pkgs/top-level/python-packages.nix

```diff
@@ -1251,7 +1251,6 @@ self: super: with self; {
   azure-mgmt-privatedns = callPackage ../development/python-modules/azure-mgmt-privatedns { };
 
   azure-mgmt-rdbms = callPackage ../development/python-modules/azure-mgmt-rdbms { };
-
```

Comment:
Why's this line?

Reply:
i didnt add this i don't think ? local `git diff master` doesn't show it..... if it is showing up here tho I can remove it

Reply:
removed it in latest push